charts/prometheus-cloudwatch-exporter/templates/deployment.yaml

```
@@ -13,6 +13,9 @@ spec:
     matchLabels:
       app: {{ template "prometheus-cloudwatch-exporter.name" . }}
       release: {{ .Release.Name }}
+{{- if .Values.deployment.matchLabels }}
+{{ toYaml .Values.deployment.matchLabels | indent 6 }}
```

You can align indentation of `if` and `toYaml` where it belongs if you use `nindent`.

```suggestion
      {{ toYaml .Values.deployment.matchLabels | nindent 6 }}
```

The same comment about indentation applies to other changes in this PR.